src/Widget/OffsetPagination.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Widget;
+
+use Closure;
+use Yiisoft\Data\Paginator\OffsetPaginator as Paginator;
+use Yiisoft\Yii\Bootstrap4\Html;
+use Yiisoft\Yii\Bootstrap4\Widget;
+
+class OffsetPagination extends Widget
```

Comment:
This one could be eventually moved to https://github.com/yiisoft/yii-dataview.

Reply:
I'm not against
But this is an implementation for Bootstrap4

Reply:
OK. Let's have it here fow now.